TestResultSummaryService/parsers/Parser.js

```diff
@@ -9,8 +9,10 @@ class Parser {
         const javaVersionRegex = /(((openjdk|java) version[\s\S]*?)(JCL.*)|((openjdk|java) version[\s\S]*?)(Server VM.*))/;
         const javaBuildDateRegex = /\s([0-9]{4})(0[1-9]|1[012])(0[1-9]|[12][0-9]|3[01])/;
         const sdkResourceRegex = /.*?SDK_RESOURCE\=(.*)[\r\n]/;
+        const jdkVersionRegex = /JDK_VERSION\=(.*)/;
+        const jvmTypeRegex = /JDK_IMPL\=(.*)/;

```

Comment:
`JDK_VERSION ` and `JDK_IMPL ` are TKG specific values. If a test does not use TKG, these values do not exist. Even with TKG, `JDK_VERSION ` and `JDK_IMPL ` are optional. Please make this generic and use regex and get this info from java version output.